packages/legend-extension-dsl-data-space/src/graph/DSL_DataSpace_HashUtils.ts

```diff
@@ -19,6 +19,8 @@ export enum DATA_SPACE_HASH_STRUCTURE {
   DATA_SPACE_EXECUTION_CONTEXT = 'DATA_SPACE_EXECUTION_CONTEXT',
   DATA_SPACE_ELEMENT_POINTER = 'DATA_SPACE_ELEMENT_POINTER',
   DATA_SPACE_EXECUTABLE = 'DATA_SPACE_EXECUTABLE',
+  DATA_SPACE_PACKAGEABLEMENT_EXECUTABLE = 'DATA_SPACE_PACKAGEABLEMENT_EXECUTABLE',
```

Comment:
Nit: I think this is a typo
```suggestion
  DATA_SPACE_PACKAGEABLE_ELEMENT_EXECUTABLE = 'DATA_SPACE_PACKAGEABLE_ELEMENT_EXECUTABLE',
```